pyproject.toml

```diff
@@ -1,12 +1,62 @@
 [build-system]
-requires = ["setuptools>=45", "wheel", "setuptools_scm>=6.2"]
+requires = ["setuptools>=45", "wheel", "setuptools_scm[toml]>=6.2"]
 build-backend = "setuptools.build_meta"
 
+[tool.setuptools_scm]
+
+[tool.setuptools.packages.find]
+include = ["quartodoc"]
+
 [tool.pytest.ini_options]
-markers = [
+markers = []
+testpaths = ["quartodoc/tests"]
+
+[project]
+name = "quartodoc"
+description = "Generate API documentation with Quarto."
+readme = "README.md"
+keywords = ["documentation", "quarto"]
+classifiers = [
+    "License :: OSI Approved :: MIT License",
+    "Operating System :: Microsoft :: Windows",
+    "Operating System :: Unix",
+    "Operating System :: MacOS",
+    "Programming Language :: Python :: 3 :: Only",
+    "Programming Language :: Python :: 3.9",
+    "Programming Language :: Python :: 3.10"
 ]
-testpaths = [
-    "quartodoc/tests"
+dynamic = ["version"]
+requires-python = ">=3.9"
+dependencies = [
+    "click",
+    "griffe==0.25.0",
+    "sphobjinv >= 2.3.1",
+    "tabulate >= 0.9.0",
+    "importlib-metadata >= 5.1.0",
+    "importlib-resources >= 5.10.2",
+    "pydantic < 2.0",
+    "pyyaml",
+    "typing-extensions >= 4.4.0",
+    "watchdog >= 3.0.0",
+    "plum-dispatch < 2.0.0; python_version < '3.10'",
+    "plum-dispatch > 2.0.0; python_version >= '3.10'"
 ]
 
-[tool.setuptools_scm]
+
+[project.urls]
```

Comment:
Do you mind if we move `project.urls`, `project.authors` and project.license` up into the project section above? (optional-dependencies and scripts can stay their own sections, since they can get fairly large).

Here is an example of how this can be done:

https://github.com/pypa/setuptools_scm/blob/main/pyproject.toml#L23

Reply:
oh cool yeah I'll make this change, makes sense

Reply:
This is done now